CLI/src/config/skeletons.xml

```diff
@@ -279,7 +285,7 @@
         @{
         var items = "";
         if (@Model.definitions.functionsGeneratedAsNested.Public.Count > 0) {
-        items += "01 PntTab-Pnt POINTER.\n";
+        items += "01 FunctionCode pic X(30).\n";
```

Comment:
Is this the same as `TC-FunctionCode` used at line 343 ?

Reply:
Currently, simplified coden does not manage nested functions. This code should be tested.